### DISCUSSION TITLE
customer portal

### DISCUSSION THREAD

**Discussion by timj257** [Labs64/NetLicensing-Community#104] [Questions]
**Question**
Ask question about NetLicensing services and functionality here.

**Additional context**
Add any other context or screenshots related to this question.

**Comment by r-brown** [comment]
Hi @timj257 - do you have any specific question reg. _Customer Portal_?
Please let us know and we'll be happy to support you.

Meanwhile you may find this [NetLicensing Customer Portal](https://netlicensing.io/wiki/customer-portal) description useful.
